lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -142,14 +145,14 @@ class UnusedBytes extends Audit {
       networkNode.record._transferSize = originalTransferSize;
     });
 
-    const savingsOnTTI = Math.max(
-      Interactive.getLastLongTaskEndTime(simulationBeforeChanges.nodeTimings) -
-        Interactive.getLastLongTaskEndTime(simulationAfterChanges.nodeTimings),
-      0
-    );
+    const savingsOnLoad = options.includeLoad ?
+      simulationBeforeChanges.timeInMs - simulationAfterChanges.timeInMs :
+      Number.MIN_VALUE;
+    const savingsOnTTI = Interactive.getLastLongTaskEndTime(simulationBeforeChanges.nodeTimings) -
+      Interactive.getLastLongTaskEndTime(simulationAfterChanges.nodeTimings);
```

Comment:
rather than this `Number.MIN_VALUE` thing, what about
```js
const savingsOnOverallLoad = simulationBeforeChanges.timeInMs - simulationAfterChanges.timeInMs;
const savingsOnTTI = ...;

const savings = options.includeLoad ? Math.max(savingsOnOverallLoad, savingsOnTTI) : savingsOnTTI;
return Math.round(savings / 10) * 10;
```
?

Reply:
sg done